CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## In Development
-
+* Add post-delete to jobs hook definition to remove them on uninstall
```

Comment:
Let's include the PR # to follow the same consistent format with the other records in the history.
```suggestion
* Add post-delete to jobs hook definition to remove them on uninstall (#161)
```

Also feel free to also include your github username as someone who contributed this fix, if you want.

Reply:
Ah, sure, I'll add it to the other PRs as well